tests/NewCypress/cypress/e2e/regression/trusts/data-download-trust.cy.ts

```diff
@@ -0,0 +1,15 @@
+import dataDownload from "../../../pages/trusts/dataDowload";
+
+describe('Trust export and content verification', () => {
+  beforeEach(() => {
+    cy.login()
+    cy.visit('/trusts/academies/details?uid=5712');
+  });
+
+  it('should export a trust and verify it has downloaded and has content', () => {
```

Comment:
Now reflected this

Reply:
This still doesn't specify anywhere that we're downloading just the academies in trust data. I wonder whether we should also make it clearer that it's a spreadsheet (I see we're doing some filtering in `findLatestFile` which we should make a little more obvious through the names of the functions within the chain)

Reply:
I have updated the text for the test to be more descriptive - I think on renaming the functions etc I'd like to keep this as is just in case we want to use it as something more reusable in the future outside of spreadsheets etc

Reply:
The problem with the function names is that right now they sound generic and would be tempting for someone to reuse without understanding that right down in the weeds it's just xlsx files that they're dealing with.

If changing the function names to reflect that it's coded to only look at spreadsheets isn't desirable then we could remove the xlsx filter so it truly is a generic file handler. Perhaps then when we have more time we can add it back in or filter on the expected file name.

I'm also happy to put this on the back burner for now so have approved the PR

Reply:
Happy to revisit this in the near future when the contacts bits etc are out of the way! I want to at some point before the framework gets too long in the tooth and do a refinement pass over and comb through bits like these and some of the other stuff we discussed recently